pkgs/applications/audio/sony-headphones-client/default.nix

```diff
@@ -7,8 +7,9 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "Plutoberth";
     repo = "SonyHeadphonesClient";
-    rev = "v${version}";
-    hash = "sha256-0DQanrglJiGsN8qQ5KxkL8I+Fpt1abeeuKiM8v9GclM=";
+    #rev = "v${version}";
+    rev = "c8f60c030bfe1987415d05241a6839648d40b358";
+    hash = "sha256-xRbPFaPlO5hyDQJyM4Pf56icRcJAL9H46Ep+hcl3DUg=";
     fetchSubmodules = true;
```

Comment:
Can we fetch this as a patch instead of touching that logic?

Reply:
I feel this is the least effort solution. Otherwise I will have to cleanup later...
(Additionally the original patch from github can not be reuesed due to the different `sourceRoot`)

Reply:
Applying a patch onto a src that uses a tag is a much more semantically valuable approach.

Reply:
seems more work, but sure. For anybody else this is easier to understand.

Reply:
@mweinelt please have a look if this is the way you wanted it.